estate/models/estate_property.py

```diff
@@ -0,0 +1,111 @@
+from odoo import models, fields,api
+from dateutil.relativedelta import relativedelta
+from datetime import date
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_is_zero, float_compare
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Test Model"
+    _order = "id desc"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=lambda self: date.today() + relativedelta(months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facade = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[
+            ('north','North'),
+            ('south','South'), 
+            ('east','East'), 
+            ('west','West')
+        ],
+    )
+    # active = fields.Boolean(default=False)
+    estate = fields.Selection(
+        string = 'Estado',
+        selection=[('new','New'),('offer received','Offer Received'),('offer accepted','Offer Accepted'),('sold','Sold'),('canceled','Canceled')],
+        required=True,
+        copy=False,
+        default='new'
+    )
+
+
+    # relation between models
+    property_type_id = fields.Many2one("estate.property.type", string="Property type" )
+
+    buyer = fields.Many2one("res.partner", string="Buyer", copy=False)
```

Comment:
Remember that when we have a relational field, if its a single record it needs to have `_id` postfix if there are many then `_ids`

```suggestion
    buyer_id = fields.Many2one("res.partner", string="Buyer", copy=False)
```